tests/test_iib_build_details_model.py

```diff
@@ -456,3 +477,20 @@ def test_merge_index_image_model_attributes(
     assert model.source_from_index_resolved == model._data["source_from_index_resolved"]
     assert model.target_index == model._data["target_index"]
     assert model.target_index_resolved == model._data["target_index_resolved"]
+
+
+def test_add_deprecations_model_attributes(fixture_add_deprecations_build_details_json):
+    model = AddDeprecationsModel.from_dict(fixture_add_deprecations_build_details_json)
+
+    assert model.binary_image == model._data["binary_image"]
+    assert model.binary_image_resolved == model._data["binary_image_resolved"]
+    assert model.deprecation_schema == model._data["deprecation_schema"]
+    assert model.from_index == model._data["from_index"]
+    assert model.from_index_resolved == model._data["from_index_resolved"]
+    assert model.index_image == model._data["index_image"]
+    assert model.internal_index_image_copy == model._data["internal_index_image_copy"]
+    assert (
+        model.internal_index_image_copy_resolved
+        == model._data["internal_index_image_copy_resolved"]
+    )
+    assert model.operator_package == model._data["operator_package"]
```

Comment:
Would it be better to perform asserts on the fixture?

Reply:
Fixtures usually prepare data for testing. And testing it self (asserts in this case) are done in test function.

Reply:
What if we would somehow (by mistake) edit `model._data` during `from_dict` method? I do not get, how does it make sense to use model._data[key], instead of fixture[key].

Reply:
> What if we would somehow (by mistake) edit model._data during from_dict method?
Then that is something we intend to catch with this test, no? We want to make sure the data is accurately assigned to attributes. 

Also, I was following suit w.r.t other tests..

Reply:
I tried several things, and right now this is not an issue, so we can mark this as resolved. This could become a problem if we had more optional arguments, and worked with them within the `from_dict` method.